packages/wescheme-blocks/src/ast.tsx

```diff
@@ -16,50 +17,79 @@ function pluralize(noun: string, set: unknown[]) {
   return set.length + " " + noun + (set.length != 1 ? "s" : "");
 }
 
+const WeSchemeAST = createASTSpec({
+  ...CoreAST.config,
+  letLikeExpr: {
+    spec: Spec.nodeSpec([
+      Spec.value<string, "form">("form"),
```

Comment:
This part is also a little bit weird. Our code expects value fields to be a certain type, which was never actually defined in the spec. With typescript, we can specify it for typescript code (it doesn't turn into a runtime check) using a generic, but then we also have to specify the field name in the type.

Reply:
Do you see this as a problem, for now? I agree that it's non-uniform compared to other specs, but I want us to leave the door open for languages to export a `typecheck : AST -> AST` function that stores type information in nodes that can be used by the `render()` function.